### PR TITLE
fix alt+enter keybinding

### PR DESCRIPTION
**Pull request by Neftedollar**
This pull request fix conflict with [new alt+enter keybinding](https://code.visualstudio.com/Updates#_editor) and fsi.SendSelection-command keybinding

### CODE REVIEW

File: release/package.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
         "command": "fsi.SendSelection",
         "key": "alt+Enter",
         "mac": "alt+Enter"
```
Comment (Krzysztof-Cieslak) [inline release/package.json]: `,` missing here at the end of line

**Comment by Krzysztof-Cieslak** [comment]
Thanks. Just one minor issue. :)

**Comment by Neftedollar** [comment]
@Krzysztof-Cieslak thanks for review, fixed.

**Comment by Krzysztof-Cieslak** [comment]
Thanks, released in `1.8.4`

**Pull request merged**